sdk/go/pulumi-language-go/main.go

```diff
@@ -353,17 +353,22 @@ func runCmdStatus(cmd *exec.Cmd, env []string) (int, error) {
 	cmd.Stdout, cmd.Stderr = os.Stdout, os.Stderr
 
 	err := cmd.Run()
+	// The returned error is nil if the command runs, has no problems copying stdin, stdout, and stderr, and
+	// exits with a zero exit status.
+	if err == nil {
```

Comment:
Bit of fun. pkg/errors.Wrap would return a nil error if the input error was nil. fmt.Errrof does not, so we now need to explicitly handle the nil case.